src/PKSim.UI/Views/Core/BaseUserControlWithValueInGrid.cs

```diff
@@ -65,14 +71,24 @@ protected virtual void RaiseButtonClick(UxGridView gridView, GridColumn column,
             edit.PerformClick(ehi.HitObject.DowncastTo<EditorButtonObjectInfoArgs>().Button);
             ((DXMouseEventArgs)e).Handled = true;
          }
-         return;
       }
 
-      protected virtual bool ColumnIsButton(GridColumn column)
+      protected virtual void RaiseCheckEditClick(UxGridView gridView, GridColumn column, int rowHandle, MouseEventArgs e)
       {
-         return false;
+         //Adapted from https://supportcenter.devexpress.com/ticket/details/t230842/grid-the-buttonclick-event-is-not-raised-immediately-when-multi-selection-is-enabled
```

Comment:
@Yuri05 I am not sure why all this is suddenly required. I hope that a new version of devexpress will fix this behavior

Reply:
we will see...